plugin/core/constants.py

```diff
@@ -0,0 +1,4 @@
+# TODO: Move all constants into this file, so that all other modules can import them without causing import loops
+
+HOVER_HIGHLIGHT_KEY = "lsp_hover_highlight"
+HOVER_PROVIDER_COUNT_KEY = "lsp_hover_provider_count"
```

Comment:
I don't think I like the idea of having all constants in a single file. I think the constants are easier to manage when they are together with relevant piece of the code.

Maybe there is a place for **some** global constants but I wouldn't make it a rule.

Reply:
Maybe we can mark all local / file-specific constants with a leading underscore and only move constants which are shared between multiple files into a common place like this?

We basically already have a part of this in `views.py`, but it's questionable to me why that file is named `views` and I think it would be better to only have imports from `.protocol` and `.typing` in the file which collects all the shared constants.